tsconfig.json

```diff
@@ -0,0 +1,13 @@
+{
+  "compilerOptions": {
+    "module": "commonjs",
+    "target": "es6",
+    "outDir": "dist",
+    "rootDir": ".",
+    "sourceMap": true,
+    "strict": false
```

Comment:
true?

Reply:
It is false in all the other projects. Do you want i set it to true for this project?

Reply:
you mean the functions? that's a mistake then, yeah I'd like to have strict checks for all our code